img/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>The Basic Language ogf the Web:HTML</title>
+        </head>
+        <body>
+            <style>
+                body {background: lightgrey;}
+                head {color:bisque;}
+                </style>
+                📘<p><h1><strong>The Code Magazine</p></h1></strong>
```

Comment:
If you want to wrap one tag into another the first opened tag should be closed the last. 
So the correct order is: 
`<h1><p>BLahblah</p></h1>`